### ISSUE TITLE
Beatmaps are playable, but have no music or background

### ISSUE THREAD

**Issue by PistolRcks** [ppy/osu#2614]
I have wanted to start using this since I finally found instructions to build it. Seeing no build errors, I started it up and dropped in an osz. Much to my dismay, there was no music or background to beatmaps, although still being fully playable. I have tried redownloading beatmaps, starting osu!lazer with the beatmap as a command-line argument, and packaging my own beatmaps and dropping them in, but to no avail. The log files don't seem to show anything, but I will include them anyway.

Screenshot of the Issue: 
![Here's a screenshot](https://user-images.githubusercontent.com/17933691/40390921-5b4f1ddc-5ddc-11e8-9a18-ba58cf7dfd17.png)

Log Files:
[network.log](https://github.com/ppy/osu/files/2028642/network.log)
[performance.log](https://github.com/ppy/osu/files/2028643/performance.log)
[runtime.log](https://github.com/ppy/osu/files/2028644/runtime.log)

Using commit 088deee on Ubuntu Budgie 18.04.

**Comment by Aergwyn** [comment]
Can you check the audio settings of the game (or even of your OS) if it's accidentally muted?

**Comment by PistolRcks** [comment]
@Aergwyn Oh, the sounds work in-game (hitsounds, menu sfx, etc.), but the music from the beatmap doesn't play.

**Comment by Aergwyn** [comment]
That includes ingame music level being above 0%?

**Comment by PistolRcks** [comment]
@Aergwyn Yes, it does. I haven't changed any audio settings (other than changing the audio device to Pulse because I thought that might work).

**Comment by LovelyA72** [comment]
It will also happen to me if I import a manually packed osz.(zip then rename)

**Comment by PistolRcks** [comment]
Although, @LovelyA72, I redownloaded Imaginary Places (which is not featured in the screenshot), and the problem occurs here as well.
